src/System.Management.Automation/CoreCLR/CorePsPlatform.cs

```diff
@@ -644,21 +988,39 @@ internal static class NativeMethods
                 [DllImport(psLib, CharSet = CharSet.Ansi)]
                 internal static extern uint GetCurrentThreadId();
 
-                // This is a struct tm from <time.h>
+                // This is a struct tm from <time.h>.
                 [StructLayout(LayoutKind.Sequential)]
                 internal unsafe struct UnixTm
                 {
-                    public int tm_sec;    /* Seconds (0-60) */
-                    public int tm_min;    /* Minutes (0-59) */
-                    public int tm_hour;   /* Hours (0-23) */
-                    public int tm_mday;   /* Day of the month (1-31) */
-                    public int tm_mon;    /* Month (0-11) */
-                    public int tm_year;   /* Year - 1900 */
-                    public int tm_wday;   /* Day of the week (0-6, Sunday = 0) */
-                    public int tm_yday;   /* Day in the year (0-365, 1 Jan = 0) */
-                    public int tm_isdst;  /* Daylight saving time */
+                    /// <summary>Seconds (0-60).</summary>
+                    internal int tm_sec;
```

Comment:
Any reason for changing these to `internal`? I know it's done elsewhere, but usually my reasoning goes like "If this type were public, should this field be visible?"